packages/node_modules/sample-browser-auth-implicit/test/wdio/specs/implicit-grant-authentication.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-undef */
+
+import testUsers from '@ciscospark/test-helper-test-users';
+import {assert} from 'chai';
+
+describe(`sample-browser-auth-implicit`, () => {
+  describe(`authenticate using implicit grant flow`, () => {
+    const browserSpock = browser.select(`browserSpock`);
+    let spock;
+
+    before(`create Spock`, () => testUsers.create({count: 1, config: {displayName: `Spock`}})
+      .then((users) => {
+        [spock] = users;
+      }));
+
+
+    step(`browse to sample app, and verify the user is not authenticated`, () => {
+      browserSpock.url(`/packages/node_modules/sample-browser-auth-implicit`);
+      assert.equal(browserSpock.getText(`#authentication-status`), `Not Authenticated`);
+    });
+
+    step(`initiate implicit grant authentication flow`, () => {
+      browserSpock.click(`[title="authenticate"]`);
+    });
+
+    step(`enter user login information`, () => {
+      // Enter Email
+      browserSpock.waitUntil(() => browser.select(`browserSpock`).getTitle() === `Sign In - Cisco Spark`);
+      browserSpock.setValue(`[placeholder="Email address"]`, spock.email);
+      browserSpock.click(`[name="btnOK"]`);
+
+      // Enter Password
+      browserSpock.waitUntil(() => browserSpock.getTitle() === `Sign In - Cisco Spark`);
+      browserSpock.setValue(`[placeholder="Password "]`, spock.password);
+      browserSpock.click(`[name="Login.Submit"]`);
+    });
+
+    step(`verify authentication was successfull`, () => {
+      browserSpock.waitUntil(() => browserSpock.getTitle() === `Sample: in Browser Authentication`);
+      browserSpock.pause(500);
+      assert.equal(browserSpock.getText(`#authentication-status`), `Authenticated`);
+      assert.equal(browserSpock.getAttribute(`[title="authenticate"]`, `value`), `logout`);
+    });
+
+    step(`logout`, () => {
+      browserSpock.click(`[title="authenticate"]`);
+      browserSpock.waitUntil(() => browser.select(`browserSpock`).getTitle() === `Sample: in Browser Authentication`);
+      browserSpock.pause(500);
```

Comment:
same as above, avoid using pause in tests if there's any thing else to trigger on.

Reply:
Sorry to pile on, but if you find you HAVE to use a pause, please put a comment above it as to why. If we're on a faster machine, having a pause will just make the tests take longer than they need to be. If we're on a slow machine, it could cause the tests to fail prematurely.